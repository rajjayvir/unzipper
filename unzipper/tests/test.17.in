0x0d
0x6b 0x10 0x00
0x6a 0x28 0x00
0x69 0x3c 0x00
0x68 0x4e 0x00
0x67 0x5f 0x00
0x0a 0x6f 0x80
0x78 0xcf 0xc0
0x61 0xcf 0xc1
0x62 0xbf 0xc2
0x63 0xaf 0xc4
0x64 0x9f 0xc8
0x65 0x8f 0xd0
0x66 0x7f 0xe0
0x00 0x00 0x10 0x3f
0xfc 0x0f 0xc1 0xfc 0x1f 0xc3 0xf8 0x7f 0x0f 0xe1 
0xfc 0x7f 0x1f 0xc7 0xf1 0xfc 0x7f 0x1f 0xc7 0xf1 
0xfc 0xfe 0x7f 0x3f 0x9f 0xcf 0xe7 0xf3 0xf9 0xfc 
0xfe 0x7f 0x3f 0x9f 0xcf 0xe7 0xf3 0xf9 0xfd 0xfd 
0xfd 0xfd 0xfd 0xfd 0xfd 0xfd 0xfd 0xfd 0xfd 0xfd 
0xfd 0xfd 0xfd 0xfd 0xfd 0xfd 0xfd 0xfd 0xfd 0xfd 
0xfd 0xfd 0xfd 0xfd 0xfd 0xfd 0xfd 0xfd 0xfd 0xfd 
0xfb 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 
0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 
0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 
0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 
0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 
0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xef 0x7b 0xde 
0xf7 0xbd 0xef 0x7b 0xde 0xf7 0xbd 0xef 0x7b 0xde 
0xf7 0xbd 0xef 0x7b 0xde 0xf7 0xbd 0xef 0x7b 0xde 
0xf7 0xbd 0xef 0x7b 0xde 0xf7 0xbd 0xef 0x7b 0xde 
0xf7 0xbd 0xef 0x7b 0xde 0xf7 0xbd 0xef 0xbd 0xef 
0x7b 0xde 0xf7 0xbd 0xef 0x7b 0xde 0xf7 0xbd 0xef 
0x7b 0xde 0xf7 0xbd 0xef 0x7b 0xde 0xf7 0xbd 0xef 
0x7b 0xde 0xf7 0xbd 0xef 0x7b 0xde 0xf7 0xbd 0xef 
0x7b 0xde 0xf7 0xbd 0xef 0x7b 0xde 0xf7 0xbe 0xee 
0xee 0xee 0xee 0xee 0xee 0xee 0xee 0xee 0xee 0xee 
0xee 0xee 0xee 0xee 0xee 0xee 0xee 0xee 0xee 0xee 
0xee 0xee 0xee 0xee 0xee 0xee 0xee 0xee 0xee 0xee 
0xee 0xfb 0xbb 0xbb 0xbb 0xbb 0xbb 0xbb 0xbb 0xbb 
0xbb 0xbb 0xbb 0xbb 0xbb 0xbb 0xbb 0xbb 0xbb 0xbb 
0xbb 0xbb 0xbb 0xbb 0xbb 0xbb 0xbb 0xbb 0xbb 0xbb 
0xbb 0xbb 0xbb 0xbb 0xee 0xee 0xee 0xee 0xee 0xee 
0xee 0xee 0xee 0xee 0xee 0xee 0xee 0xee 0xee 0xee 
0xee 0xee 0xee 0xee 0xee 0xee 0xee 0xee 0xee 0xee 
0xee 0xee 0xee 0xee 0xee 0xee 0xef 0xbb 0xbb 0xbb 
0xbb 0xbb 0xbb 0xbb 0xbb 0xbb 0xbb 0xbb 0xbb 0xbb 
0xbb 0xbb 0xbb 0xbb 0xbb 0xbb 0xbb 0xbb 0xbb 0xbb 
0xbb 0xbb 0xbb 0xbb 0xbb 0xbb 0xbb 0xbb 0xbb 0xbe 
0xdb 0x6d 0xb6 0xdb 0x6d 0xb6 0xdb 0x6d 0xb6 0xdb 
0x6d 0xb6 0xdb 0x6d 0xb6 0xdb 0x6d 0xb6 0xdb 0x6d 
0xb6 0xdb 0x6d 0xb6 0xfb 0x6d 0xb6 0xdb 0x6d 0xb6 
0xdb 0x6d 0xb6 0xdb 0x6d 0xb6 0xdb 0x6d 0xb6 0xdb 
0x6d 0xb6 0xdb 0x6d 0xb6 0xdb 0x6d 0xb6 0xdb 0xed 
0xb6 0xdb 0x6d 0xb6 0xdb 0x6d 0xb6 0xdb 0x6d 0xb6 
0xdb 0x6d 0xb6 0xdb 0x6d 0xb6 0xdb 0x6d 0xb6 0xdb 
0x6d 0xb6 0xdb 0x6f 0xb6 0xdb 0x6d 0xb6 0xdb 0x6d 
0xb6 0xdb 0x6d 0xb6 0xdb 0x6d 0xb6 0xdb 0x6d 0xb6 
0xdb 0x6d 0xb6 0xdb 0x6d 0xb6 0xdb 0x6d 0xbe 0xdb 
0x6d 0xb6 0xdb 0x6d 0xb6 0xdb 0x6d 0xb6 0xdb 0x6d 
0xb6 0xdb 0x6d 0xb6 0xdb 0x6d 0xb6 0xdb 0x6d 0xb6 
0xdb 0x6d 0xb6 0xfb 0x6d 0xb6 0xdb 0x6d 0xb6 0xdb 
0x6d 0xb6 0xdb 0x6d 0xb6 0xdb 0x6d 0xb6 0xdb 0x6d 
0xb6 0xdb 0x6d 0xb6 0xdb 0x6d 0xb6 0xdb 0xed 0xb6 
0xdb 0x6d 0xb6 0xdb 0x6d 0xb6 0xdb 0x6d 0xb6 0xdb 
0x6d 0xb6 0xdb 0x6d 0xb6 0xdb 0x6d 0xb6 0xdb 0x6d 
0xb6 0xdb 0x6f 0xb6 0xdb 0x6d 0xb6 0xdb 0x6d 0xb6 
0xdb 0x6d 0xb6 0xdb 0x6d 0xb6 0xdb 0x6d 0xb6 0xdb 
0x6d 0xb6 0xdb 0x6d 0xb6 0xdb 0x6d 0xbe 0xaa 0xaa 
0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 
0xaa 0xaa 0xaa 0xaa 0xfa 0xaa 0xaa 0xaa 0xaa 0xaa 
0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 
0xab 0xea 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 
0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaf 0xaa 0xaa 
0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 
0xaa 0xaa 0xaa 0xaa 0xbe 0xaa 0xaa 0xaa 0xaa 0xaa 
0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 
0xaa 0xfa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 
0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xab 0xea 0xaa 
0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 
0xaa 0xaa 0xaa 0xaa 0xaf 0xaa 0xaa 0xaa 0xaa 0xaa 
0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 
0xaa 0xbe 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 
0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xfa 0xaa 
0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 
0xaa 0xaa 0xaa 0xaa 0xab 0xea 0xaa 0xaa 0xaa 0xaa 
0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 
0xaa 0xaf 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 
0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xbe 0xaa 
0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 
0xaa 0xaa 0xaa 0xaa 0xaa 0xfa 0xaa 0xaa 0xaa 0xaa 
0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 
0xaa 0xab 0xea 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 
0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaf 0xaa 
0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 0xaa 
0xaa 0xaa 0xaa 0xaa 0xaa 0xbe 0x00 0x00 0x00 0x00 
0x00 0x00 0x00 0x00 0xf8 0x00 0x00 0x00 0x00 0x00 
0x00 0x00 0x03 0xe0 0x00 0x00 0x00 0x00 0x00 0x00 
0x00 0x0f 0x80 0x00 0x00 0x00 0x00 0x00 0x00 0x00 
0x3e 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0xf8 
0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x03 0xe0 0x00 
0x00 0x00 0x00 0x00 0x00 0x00 0x0f 0x80 0x00 0x00 
0x00 0x00 0x00 0x00 0x00 0x3e 0x00 0x00 0x00 0x00 
0x00 0x00 0x00 0x00 0xf8 0x00 0x00 0x00 0x00 0x00 
0x00 0x00 0x03 0xe0 0x00 0x00 0x00 0x00 0x00 0x00 
0x00 0x0f 0x80 0x00 0x00 0x00 0x00 0x00 0x00 0x00 
0x3e 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0xf8 
0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x03 0xe0 0x00 
0x00 0x00 0x00 0x00 0x00 0x00 0x0f 0x80 0x00 0x00 
0x00 0x00 0x00 0x00 0x00 0x3e 0x00 0x00 0x00 0x00 
0x00 0x00 0x00 0x00 0xf8 0x00 0x00 0x00 0x00 0x00 
0x00 0x00 0x03 0xe0 0x00 0x00 0x00 0x00 0x00 0x00 
0x00 0x0f 0x80 0x00 0x00 0x00 0x00 0x00 0x00 0x00 
0x3e 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0xf8 
0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x03 0xe0 0x00 
0x00 0x00 0x00 0x00 0x00 0x00 0x0f 0x80 0x00 0x00 
0x00 0x00 0x00 0x00 0x00 0x3e 0x00 0x00 0x00 0x00 
0x00 0x00 0x00 0x00 0xf8 0x00 0x00 0x00 0x00 0x00 
0x00 0x00 0x03 0xe0 0x00 0x00 0x00 0x00 0x00 0x00 
0x00 0x0f 0x80 0x00 0x00 0x00 0x00 0x00 0x00 0x00 
0x3e 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0xf8 
0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x03 0xe0 0x00 
0x00 0x00 0x00 0x00 0x00 0x00 0x0f 0x80 0x00 0x00 
0x00 0x00 0x00 0x00 0x00 0x3e 