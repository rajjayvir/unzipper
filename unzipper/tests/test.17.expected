xaabbbbccccccccddddddddddddddddeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee
ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff
gggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggg
gggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggg
hhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhh
hhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhh
hhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhh
hhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhh
iiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiii
iiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiii
iiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiii
iiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiii
iiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiii
iiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiii
iiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiii
iiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiii
jjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjj
jjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjj
jjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjj
jjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjj
jjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjj
jjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjj
jjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjj
jjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjj
jjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjj
jjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjj
jjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjj
jjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjj
jjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjj
jjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjj
jjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjj
jjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjj
kkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkk
kkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkk
kkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkk
kkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkk
kkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkk
kkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkk
kkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkk
kkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkk
kkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkk
kkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkk
kkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkk
kkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkk
kkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkk
kkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkk
kkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkk
kkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkk
kkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkk
kkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkk
kkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkk
kkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkk
kkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkk
kkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkk
kkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkk
kkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkk
kkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkk
kkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkk
kkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkk
kkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkk
kkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkk
kkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkk
kkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkk
kkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkk
